gammapy/data/data_store.py

```diff
@@ -201,24 +198,24 @@ def from_events_files(cls, events_paths, irfs_paths=None):
         - ``IRF`` (example: ``IRF = South_z20_50h``)
 
         This method is useful specifically if you want to load data simulated
-        with `ctobssim`_
+        with `ctobssim`_.
```

Comment:
```suggestion
        with `ctobssim`.
```

Reply:
The underscore has to stay here, it is to create the link that you can see here : https://docs.gammapy.org/dev/api/gammapy.data.DataStore.html#gammapy.data.DataStore.from_events_files